src/mavo.js

```diff
@@ -177,7 +177,14 @@ var _ = self.Mavo = $.Class({
 
 		this.permissions.can("login", () => {
 			// We also support a URL param to trigger login, in case the user doesn't want visible login UI
-			if (Mavo.Functions.url("login") !== null && this.index == 1 || Mavo.Functions.url(this.id + "-login") !== null) {
+			if (Mavo.Functions.url("login") !== null && this.index === 1) {
+				// Remove ?login from url
+				const currentURL = new URL(location.href);
+				currentURL.searchParams.delete("login");
+				history.replaceState(null, "", currentURL.toString());
+				this.primaryBackend.login();
+			}
+			else if (Mavo.Functions.url(this.id + "-login") !== null) {
```

Comment:
Why not do the same with this param? It's the same thing, it's just that instead of `login`, it's prepended with the app name, to target a specific Mavo app on the page.

Reply:
I see; didn't realize what you meant by "There is also a more specific one, for cases when there are multiple Mavo apps on the page." earlier but this makes sense now